jdwp/common.py

```diff
@@ -0,0 +1,61 @@
+"""Basic types for JDWP messages."""
+
+from typing import List
+
+
+class Types:
+    """Types class."""
+
+    STRING = str
+    INT = int
+    BYTE = float
+    BOOLEAN = bool
+    DICT = dict
+    REFERENCE_TYPE_ID = "referenceTypeID"
+    CLASS_LOADER = "classLoader"
+    FIELD_ID = "fieldID"
+    METHOD_ID = "methodID"
+    VALUE = "value"
+    INTERFACE_ID = "interfaceID"
+    CLASS_OBJECT_ID = "classObjectID"
+    TAGGED_OBJECT_ID = "taggedObjectID"
+    THREAD_ID = "threadID"
+    THREAD_GROUP_ID = "threadGroupID"
+    OBJECT_ID = "objectID"
+    LOCATION = "location"
+
+
+class Field:
```

Comment:
It might be more convenient to use `NamedTuple`s or dataclasses here and later. They are immutable, less verbose to declare and have whole bunch of useful methods out of the box:

```
class Field(typing.NamedTuple):
    name: str
    type: Type
    description: str
```